taichi/codegen/dx12/codegen_dx12.cpp

```diff
@@ -232,7 +232,15 @@ KernelCodeGenDX12::CompileResult KernelCodeGenDX12::compile() {
   TI_AUTO_PROF;
   const auto &config = get_compile_config();
 
-  irpass::ast_to_ir(config, *kernel, false);
+  bool verbose = config.print_ir;
+  if (kernel->is_accessor && !config.print_accessor_ir) {
+    verbose = false;
```

Comment:
do we use this `verbose` variable btw?

Reply:
Sorry 😂, Update: https://github.com/taichi-dev/taichi/pull/7765/commits/468ef034544daec0ae78292a3ab12cd8c8d89485